crates/bevy_internal/Cargo.toml

```diff
@@ -71,17 +71,18 @@ shader_format_glsl = [
 shader_format_spirv = ["bevy_render/shader_format_spirv"]
 
 serialize = [
+  "bevy_color?/serialize",
   "bevy_core/serialize",
-  "bevy_input/serialize",
   "bevy_ecs/serialize",
-  "bevy_time/serialize",
-  "bevy_window/serialize",
-  "bevy_winit?/serialize",
-  "bevy_transform/serialize",
+  "bevy_input/serialize",
   "bevy_math/serialize",
   "bevy_scene?/serialize",
+  "bevy_sprite?/serialize",
+  "bevy_time/serialize",
+  "bevy_transform/serialize",
   "bevy_ui?/serialize",
-  "bevy_color?/serialize",
+  "bevy_window/serialize",
+  "bevy_winit?/serialize",
```

Comment:
Since the order seemed mostly random before, I decided to sort these. The big change is that `bevy_sprite?/serialize` is added here.